wiki/osu!academy/en.md

```diff
@@ -1,87 +1,91 @@
-| ![](img/academy-23.png) | [Episode 23](https://www.youtube.com/watch?v=MxlB__wjt9A "YouTube") | In this episode, we will be talking about a new feature that has recently been sweeping over the modding community and that is community modding featuring Shiro! |
-| ![](img/academy-24.png) | [Episode 24](https://www.youtube.com/watch?v=pq33jvMitRk "YouTube") | In this episode, we will be going over how to improve with streaming featuring Kyonko Hizara! |
-| ![](img/academy-25.png) | [Episode 25](https://www.youtube.com/watch?v=sgcdrxevAT4 "YouTube") | In this episode, we are going to be talking about the staff teams a bit more! |
-| ![](img/academy-26.png) | [Episode 26](https://www.youtube.com/watch?v=y61v2QCHlpY "YouTube") | The following episode of the osu! academy is brought to you by a public service announcement! |
-| ![](img/academy-27.png) | [Episode 27](https://www.youtube.com/watch?v=nXWA1Qh9bT8 "YouTube") | In this episode, we will go over how to make and share your own guest difficulties! |
-| ![](img/academy-28.png) | [Episode 28](https://www.youtube.com/watch?v=PEZFOM8NKtw "YouTube") | In this episode, we will cover forum basics such as where to post and correct behaviour! |
-| ![](img/academy-29.png) | [Episode 29](https://www.youtube.com/watch?v=uvCRwcyJ4TA "YouTube") | In this episode, we will cover the basics of storyboarding! |
-| ![](img/academy-30.png) | [Episode 30](https://www.youtube.com/watch?v=EvICgPuOylk "YouTube") | In this episode, we will cover more advanced storyboarding! |
-| ![](img/academy-31.png) | [Episode 31](https://www.youtube.com/watch?v=s2ZK4o8V5tI "YouTube") | In this episode, we will go further in-depth about guest difficulties and other beatmap contributions! |
-| ![](img/academy-32.png) | [Episode 32](https://www.youtube.com/watch?v=wa_hNegtBw0 "YouTube") | In this episode, we will go over the new ranking system! |
-| ![](img/academy-33.png) | [Episode 33](https://www.youtube.com/watch?v=_-Xke2bqzok "YouTube") | In this episode, we will cover more advanced editor tips! |
+*Note: Episodes 1-6 have been rerecorded, as they presented obsolete information, and thus were deemed as unhelpful.*
+
+| No. | Title | Description |
+| :-: | :-- | :-- |
+| 0 | [Lazer installer setup guide](https://www.youtube.com/watch?v=kO5ISAzRbPE "YouTube") | [ztrot](https://osu.ppy.sh/users/6347) covers the process of installing the osu!lazer client. |
+| 1 | [osu! Installation/Account Setup/Beatmap Downloads](https://www.youtube.com/watch?v=0V5GwzmMhpU "YouTube") | [pishifat](https://osu.ppy.sh/users/3178418) explains how to install osu!, create an account, and download new beatmaps. |
+| 2 | [Standard Gameplay/Settings Adjustments](https://www.youtube.com/watch?v=mswLEXK0eDk "YouTube") | [ztrot](https://osu.ppy.sh/users/6347) goes over the osu! mode, and explains how to adjust gameplay settings. |
```

Comment:
I think most of them are oxford commas so i will leave them as they are

also I would leave `mode` instead of `game mode` because it sounds more natural for me